lib/rspec/rails/example/view_example_group.rb

```diff
@@ -149,6 +149,22 @@ def _include_controller_helpers
 
         helper(*_default_helpers)
 
+        app = ::Rails.application
+        if app.respond_to?(:routes)
```

Comment:
To be honest, I was initially going with just doing a straight `Rails.applicaiton.routes.url_helpers`. However, then I saw [this code](https://github.com/rspec/rspec-rails/blob/b4c00c45cfeb4ba5be3aeef5fd2b4205f77c0a47/lib/rspec/rails/example/feature_example_group.rb#L13-L21) in `feature_example_group.rb` is doing this defensive coding, so I just copied and pasted it. I saw it was introduced in https://github.com/rspec/rspec-rails/commit/0fda98b77b43d785e095b6f525840f7d233d9a1d#diff-07cac43fac355c3ef9dac2caf3a08574R11 by @jnicklas and @alindeman, so maybe they will have some insight if it's necessary.